contracts/NodeDelegatorETH.sol

```diff
@@ -0,0 +1,411 @@
+
+            emit Undelegate(strategy, strategyShares);
+        }
+
+        delegationManager.undelegate(address(this));
+    }
+
+    /// @dev Triggers stopped state. Contract must not be paused.
+    function pause() external onlyLRTManager {
+        _pause();
+    }
+
+    /// @dev Returns to normal state. Contract must be paused
+    function unpause() external onlyLRTAdmin {
+        _unpause();
+    }
+
+    /// @dev Approves the SSV Network contract to transfer SSV tokens for deposits
+    function approveSSV() external onlyLRTManager {
+        address SSV_TOKEN_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_TOKEN);
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        IERC20(SSV_TOKEN_ADDRESS).approve(SSV_NETWORK_ADDRESS, type(uint256).max);
+    }
+
+    /// @dev Deposits more SSV Tokens to the SSV Network contract which is used to pay the SSV Operators
+    function depositSSV(uint64[] memory operatorIds, uint256 amount, Cluster memory cluster) external onlyLRTManager {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).deposit(address(this), operatorIds, amount, cluster);
+    }
+
+    // @dev Withdraws SSV Tokens from a SSV Cluster
+    function withdrawSSV(uint64[] memory operatorIds, uint256 amount, Cluster memory cluster) external onlyLRTManager {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).withdraw(operatorIds, amount, cluster);
+    }
+
+    /// @dev Registers a new validator in the SSV Cluster
+    function registerSsvValidator(
+        bytes calldata publicKey,
+        uint64[] calldata operatorIds,
+        bytes calldata sharesData,
+        uint256 amount,
+        Cluster calldata cluster
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).registerValidator(publicKey, operatorIds, sharesData, amount, cluster);
+    }
+
+    /// @dev Initiates exits from validators in a SSV Cluster
+    /// The staked ETH will eventually swept to the EigenPod.
+    /// Only the Operator can call this function.
+    /// @param publicKeys Array of validator public keys
+    /// @param operatorIds The operator IDs of the SSV Cluster
+    function exitSsvValidators(
+        bytes[] calldata publicKeys,
+        uint64[] calldata operatorIds
+    )
+        external
+        onlyLRTOperator
+        whenNotPaused
+    {
+        address SSV_NETWORK_ADDRESS = lrtConfig.getContract(LRTConstants.SSV_NETWORK);
+
+        ISSVNetwork(SSV_NETWORK_ADDRESS).bulkExitValidator(publicKeys, operatorIds);
+    }
+
+    /// @dev Remove validators from the SSV Cluster.
+    /// Make sure `exitSsvValidators` is called before and all the validators have exited the Beacon chain.
```

Comment:
I notice we have $BAD_THING and no contract code to prevent us from doing $BAD_THING. I guess since this exit is a one time product sunset, this is okay (as long as the penalty is not a full slash), and we can get our funds out.